src/main/scala/beam/utils/data/synthpop/PumaLevelScenarioGenerator.scala

```diff
@@ -59,6 +59,7 @@ class PumaLevelScenarioGenerator(
   private val congestionLevelData: CsvCongestionLevelData = new CsvCongestionLevelData(pathToCongestionLevelDataFile)
 
   private val planElementTemplate: PlanElement = PlanElement(
+    tripId = "None",
```

Comment:
I think it is better to use empty string for no-value, or, event better, to use Option[String] as field type and use None